react-web/src/components/App.js

```diff
@@ -25,12 +26,16 @@ class App extends Component {
     return (
       <div className="App">
         <NavBar />
-        <Switch>
-          {this.state.sessionId && <Redirect exact from='/' to={`/session/${this.state.sessionId}`} />}
-          <Route exact path="/" render={() => <CreateSession onSessionId={this.onSessionIdCreated} />} />
-          <Route exact path="/session/:id" component={Session} />
-          <Redirect to="/" />
-        </Switch>
+
+        <div className="Container">
+          <Switch>
+            {this.state.sessionId && <Redirect exact from='/' to={`/session/${this.state.sessionId}`} />}
+            <Route exact path="/" render={() => <CreateSession onSessionId={this.onSessionIdCreated} />} />
+            <Route exact path="/jira" component={JiraLogin} />
```

Comment:
Will this will be removed for a tab once https://github.com/maleck13/storypoint/pull/14 lands?